src/main/java/duke/Todo.java

```diff
@@ -0,0 +1,12 @@
+package duke;
```

Comment:
Minor style issue here, but there should always be a new line after package statement